src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/column/main.js

```diff
@@ -80,6 +70,90 @@ define(['sulucontent/components/open-ghost-overlay/main'], function(OpenGhost) {
                     '<div id="wait-container" style="margin-top: 50px; margin-bottom: 200px; display: none;"></div>'
                 ].join('');
             }
+        },
+
+
+        /**
+         * Enabler for selected items
+         * @param column
+         * @returns {boolean}
+         */
+        hasSelectedEnabler = function(column) {
+            return !!column.hasSelected;
+        },
+
+        /**
+         * Enabler for DELETE
+         * @param column
+         * @returns {boolean}
+         */
+        deleteEnabler = function(column) {
+            return hasSelectedEnabler(column) && SecurityChecker.hasPermission(column.selectedItem, 'delete');
+        },
+
+        /**
+         * Enabler for MOVE
+         * @param column
+         * @return {boolean}
+         */
+        moveEnabler = function(column) {
+            return hasSelectedEnabler(column) && SecurityChecker.hasPermission(column.selectedItem, 'edit');
+        },
+
+        /**
+         * Enabler for COPY
+         * @param column
+         * @returns {boolean}
+         */
+        copyEnabler = function(column) {
+            return hasSelectedEnabler(column) && SecurityChecker.hasPermission(column.selectedItem, 'view');
+        },
+
+        /**
+         * Enaber for ORDER
+         * @param column
+         * @returns {boolean}
+         */
+        orderEnabler = function(column) {
+            var editChildrenPermission = true;
+
+            $.each(column.children, function(id, childItem) {
```

Comment:
use our cool guy loop implementation in util service `services/husky/util` `uril.foreach` is much faster than `$.each` http://www.sitepoint.com/speed-question-jquery-each-vs-loop/